### PR TITLE
Fix Az.KeyVault Syntax Errors

### PR DESCRIPTION
**Pull request by v-yuzhichen**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/KeyVault/KeyVault/help/Update-AzKeyVaultSecret.md [hunk 1]
```diff
@@ -63,7 +63,9 @@ ContosoVault, using the stored variables.
 
 ### Example 2: Delete the tags and content type for a secret
 ```powershell
-Update-AzKeyVaultSecret -VaultName 'ContosoVault' -Name 'HR' -Version '9EEA45C6EE50490B9C3176A80AC1A0DF' -ContentType '' -Tag -@{}
+$Tags = @{ 'Severity' = 'medium'; 'HR' = 'true'}
+$ContentType= 'xml'
+Update-AzKeyVaultSecret -VaultName 'ContosoVault' -Name 'HR' -Version '9EEA45C6EE50490B9C3176A80AC1A0DF' -ContentType $ContentType -Tag $Tags
```
Comment (ziyuezh576) [inline src/KeyVault/KeyVault/help/Update-AzKeyVaultSecret.md:68]: -Tag @{} (delete the dash before @)

No other errors here. The purpose of this command is to delete, so blank value is assigned.

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

File: src/KeyVault/KeyVault/help/Export-AzKeyVaultSecurityDomain.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Exports the security domain data of a managed HSM for importing on another HSM.
 
 ### Example 1
 ```powershell
-Export-AzKeyVaultSecurityDomain -Name testmhsm -Certificates {pathOfCertificates}/sd1.cer, {pathOfCertificates}/sd2.cer, {pathOfCertificates}/sd3.cer -OutputPath {pathOfOutput}/sd.ps.json -Quorum 2
+Export-AzKeyVaultSecurityDomain -Name testmhsm -Certificates {{pathOfCertificates}/sd1.cer, {pathOfCertificates}/sd2.cer, {pathOfCertificates}/sd3.cer} -OutputPath {{pathOfOutput}/sd.ps.json} -Quorum 2
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/help/Export-AzKeyVaultSecurityDomain.md:36]: In the original example, `{}` is not powershell gramma. It means that `pathOfCertificates` is a placeholder that user needs to replace with the real path.
If `{}` made our tooling think the example is wrong, I suggest replace `{pathOfCertificates}/sd1.cer` to `./sd1`, and `{pathOfOutput}/sd.ps.json` to `sd.ps.json`.

File: src/KeyVault/KeyVault/help/Import-AzKeyVaultSecurityDomain.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ This cmdlet imports previously exported security domain data to a managed HSM.
 ### Example 1
 ```powershell
 $keys = @{PublicKey = "sd1.cer"; PrivateKey = "sd1.key"}, @{PublicKey = "sd2.cer"; PrivateKey = "sd2.key"}, @{PublicKey = "sd3.cer"; PrivateKey = "sd3.key"}
-Import-AzKeyVaultSecurityDomain -Name testmhsm -Keys $keys -SecurityDomainPath {pathOfBackup}\sd.ps.json
+Import-AzKeyVaultSecurityDomain -Name testmhsm -Keys $keys -SecurityDomainPath {{pathOfBackup}\sd.ps.json}
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/help/Import-AzKeyVaultSecurityDomain.md:37]: Same comment

File: src/KeyVault/KeyVault/help/Update-AzKeyVaultSecret.md (hunk 1 above)
Comment (isra-fel) [inline src/KeyVault/KeyVault/help/Update-AzKeyVaultSecret.md:68]:
```suggestion
$ContentType= 'xml'
Update-AzKeyVaultSecret -VaultName 'ContosoVault' -Name 'HR' -Version '9EEA45C6EE50490B9C3176A80AC1A0DF' -ContentType $ContentType -Tag @{}
```

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Review by ziyuezh576 — APPROVED** [review]

File: src/KeyVault/KeyVault/help/Export-AzKeyVaultSecurityDomain.md (hunk 2 above)
Comment (ziyuezh576) [reply src/KeyVault/KeyVault/help/Export-AzKeyVaultSecurityDomain.md]: Thanks. We will do that.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**